cognite/neat/graph/loaders/_rdf2asset.py

```diff
@@ -187,10 +285,33 @@ def _get_required_capabilities(self) -> list[Capability]:
                     AssetsAcl.Action.Read,
                 ],
                 scope=AssetsAcl.Scope.DataSet([self.data_set_id]),
-            )
+            ),
+            RelationshipsAcl(
+                actions=[
+                    RelationshipsAcl.Action.Write,
+                    RelationshipsAcl.Action.Read,
+                ],
+                scope=RelationshipsAcl.Scope.DataSet([self.data_set_id]),
+            ),
         ]
 
     def _upload_to_cdf(
+        self,
+        client: CogniteClient,
+        items: list[AssetWrite] | list[RelationshipWrite],
+        dry_run: bool,
+        read_issues: NeatIssueList,
+    ) -> Iterable[UploadResult]:
+        if isinstance(items[0], AssetWrite) and all(isinstance(item, type(items[0])) for item in items):
```

Comment:
Why look up the type of the first item, you already know it is an Asset
```suggestion
        if isinstance(items[0], AssetWrite) and all(isinstance(item, AssetWrite) for item in items):
```